components/sensor/ade7953.rst

```diff
@@ -85,39 +77,54 @@ Additionally, the device has an ::doc:`NTC temperature sensor <ntc>`.
 
     sensor:
       - platform: ade7953
-        irq_pin: GPIO16
+        irq_pin: GPIO16 # Prevent overheating by setting this
+        update_interval: 10s
```

Comment:
This comment is on the wrong line:
```suggestion
        irq_pin: GPIO16
        update_interval: 10s # Prevent overheating by setting this
```

Reply:
Actually it's not.  Setting `irq_pin` prevents the chip from overheating as described by [the ade7953 documentation](https://esphome.io/components/sensor/ade7953.html) and very much the case for my devices.
I will remove `update_interval` as that's just a setting

Reply:
Ah, didn't know that, then this is fine. I think `update_interval` is sufficiently easy to understand and universal that it can remain.